File: fixtures/node_output/post_with_data_binary.js
var request = require('request');

var dataString = '{"title":"china1"}';

var options = {
    url: 'http://example.com/post',
    method: 'POST',
    body: dataString
};

function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}

request(options, callback);

File: fixtures/elixir_output/post_with_data_binary.ex
method = :post
url = "http://example.com/post"
headers = []
body = ~s|{"title":"china1"}|
options = []
params = []

request = %HTTPoison.Request{
  method: method,
  url: url,
  body: body,
  headers: headers,
  options: options,
  params: params,
}

response = HTTPoison.request(request)
